frontend/packages/ux-editor/src/components/config/editModal/EditDataModelBindings.test.tsx

```diff
@@ -1,12 +1,14 @@
 import React from 'react';
-import { screen } from '@testing-library/react';
-
+import { act, screen, within } from '@testing-library/react';
 import { renderWithMockStore } from '../../../testing/mocks';
 import { appDataMock, textResourcesMock } from '../../../testing/stateMocks';
 import { IAppDataState } from '../../../features/appData/appDataReducers';
 import { EditDataModelBindings } from './EditDataModelBindings';
 import { textMock } from '../../../../../../testing/mocks/i18nMock';
 import { ComponentType } from 'app-shared/types/ComponentType';
+import userEvent from '@testing-library/user-event';
+
+const user = userEvent;
```

Comment:
Should we invoke the setup method here? 😊 
```suggestion
const user = userEvent.setup();
```

Reply:
I used userEvent instead,  no longer needs to user.